src/htmlContent/video-view.html

```diff
@@ -0,0 +1,22 @@
+<div class="video-view">
+    <div class="video-data"></div>
+    <div class="video-wrapper">
+        <video controls>
+          <source src="{{ videoUrl }}" type="video/mp4">
+        </video>
+    </div>
+    <p>Embed Code</p>
+<pre>
```

Comment:
In 2017 I don't think we need this anymore: http://caniuse.com/#search=video

However, I do think we should give some thought to how to communicate the issues around video encoding types not being supported.  Let's get this working, then loop back.